### ISSUE TITLE
Service invoked too many times in a short time

### ISSUE THREAD

**Issue by kxbin** [Blacksmoke16/GESI#99]
markets_region_orders(10000002,"all")

return 
```
Error
Exception: Service invoked too many times in a short time: urlfetch. Try Utilities.sleep(1000) between calls. (line 43).
```

**Comment by Blacksmoke16** [comment]
This is a limitation of Google Sheets in that you're limited to 25k requests per day across all of your sheets. Depending on your use case, I'd look into using an aggregated API like https://market.fuzzwork.co.uk/api/ to get your price information in a more efficient manner.

Otherwise you should also make sure you're not running that function on a schedule, and also limiting when it runs. I.e. so it doesnt do it every time you refresh the page.

In short, there's not really anything I can do about this, so it's on you to make your sheet more efficient, or switch to using an actual programming language to handle getting the data for you.
